apps/utils.py

```diff
@@ -77,3 +81,35 @@ def default(self, o):  # pylint: disable=method-hidden
         if isinstance(o, decimal.Decimal):
             return float(o)
         return super(DecimalEncoder, self).default(o)
+
+
+def get_s3_client():
```

Comment:
Let's move all of this to conf/s3.py

Reply:
Also then you should be able to just access the client like `settings.S3_CLIENT`

Reply:
While moving this `conf/s3.py`, it triggered the `@iot_mock` decorator in `test_shipments.py` to fail, due to `boto3` being setup up in the hierarchy.